app/policies/application_policy.rb

```diff
@@ -53,23 +53,11 @@ def mine?
   end
 
   # Default access scope for querying records.
-  class Scope
-    attr_reader :current_profile, :user, :scope
-
-    def initialize(user, scope)
-      @user = user
-      @current_profile = user&.profile
-      @scope = scope
-    end
-
+  Scope = Data.define(:user, :scope) do
```

Comment:
What is Data? I've never seen this before.

Reply:
It's a newer Ruby feature, similar to Struct, but immutable.

Reply:
Forgot to add a doc link! https://rubyapi.org/3.3/o/data

Reply:
Hi! Random visit due to social media and things. Just wanted to say this is neat.